### PR TITLE
Update Biome

### PR DESCRIPTION
**Pull request by nhedger**
image

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

This PR updates the logo of **Biome** to the new one.

Reference: https://github.com/biomejs/resources

### CODE REVIEW

**Comment by PeterShaggyNoble** [comment]
Thanks for the contribution, @nhedger 👍🏻

Labelling this as https://github.com/simple-icons/simple-icons/labels/pending as they have not yet rolled this new branding out to [their website](https://biomejs.dev/).

To make sure we're comparing like with like in our review, could you let us know which file you based your icon on?

**Comment by nhedger** [comment]
Thanks @PeterShaggyNoble,

The icon is based on https://github.com/biomejs/resources/blob/main/svg/icon-light-mono.svg

We've started rolling out the new logo on the GitHub org, IDE extensions, Discord, etc.

If you'd prefer for the website redesign to land to update the icon, I understand.

Cheers

File: _data/simple-icons.json [hunk 1]
```diff
@@ -1894,7 +1894,7 @@
         },
         {
             "title": "Biome",
-            "hex": "F7B911",
+            "hex": "60A5FA",
             "source": "https://biomejs.dev",
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:1898]:
```suggestion
            "source": "https://github.com/biomejs/resources/blob/551f36498dfe34b24bc7755fcdd0fa501b757cf4/svg/icon-light-mono.svg",
```

**Review by PeterShaggyNoble — APPROVED** [review]
Sorry, @nhedger, I didn't realise you were a member of the Biome team. In that case we can update our icon now on your say-so.

**Comment by nhedger** [comment]
My bad, I forgot to mention it!

Thanks for merging!

**Pull request merged**